docs/contributing/setup.md

```diff
@@ -13,9 +13,33 @@ To compile and run the tests use the following command:
 ./gradlew check
 ```
 
-## Launch it with Nextflow
+## Launch it with installed Nextflow
 
-To test with Nextflow for development purpose:
+!!! warning
+
+    This method will add the development version of the plugin to your Nextflow plugins
+    Take care when using this method and make sure that you are never using a
+    development version to run real pipelines.
+    You can delete all `nf-schema` versions using this command:
+    ```bash
+    rm -rf ~/.nextflow/plugins/nf-schema*
+    ```
+
+Install the current version of the plugin in your `.nextflow/plugins` folder
```

Comment:
Does it have to be installed with `nextflow plugin install nf-schema`? Or by clonning the repo?

Reply:
You can do this using `make install`. I'll try and make it a bit more clear that the code block underneath belongs with that piece of text